UnityProject/Assets/Scripts/Antagonists/Objectives/CargoEliminateSecurity.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System.Linq;
+
+namespace Antagonists
+{
+	[CreateAssetMenu(menuName="ScriptableObjects/Objectives/CargoEliminateSecurity")]
+	public class CargoEliminateSecurity : Objective
+	{
+		protected override void Setup()
+		{
+		}
+
+		protected override bool CheckCompletion()
+		{
+			foreach (Transform t in GameManager.Instance.PrimaryEscapeShuttle.MatrixInfo.Objects.transform)
+			{
+				var player = t.GetComponent<PlayerScript>();
+				if (player != null)
+				{
+					var playerDetails = PlayerList.Instance.Get(player.gameObject);
+					if (playerDetails.Job == JobType.SECURITY_OFFICER || playerDetails.Job == JobType.HOS
+					                                           || playerDetails.Job != JobType.DETECTIVE
+					                                           || playerDetails.Job != JobType.WARDEN)
```

Comment:
Is this right? If the playerDetails.Job != JobType.DETECTIVE or JobType.WARDEN, that should include everyone on the station, except the detective or the warden.

Reply:
thank you for pointing this out, I hadn't had much sleep and was coding on autopilot. Will fix it today